languages/julia/exercises/concept/boolean-logic/runtests.jl

```diff
@@ -0,0 +1,62 @@
+using Test
```

Comment:
The test cases are copied 1:1 from the JS exercise it was forked from. I have not verified that they work as intended.

Reply:
The example code passes the tests at least. I think it's pretty much certain that they're fine.